app/javascript/dashboard/routes/dashboard/commands/commandBarBusEvents.js

```diff
@@ -14,6 +14,10 @@ export const CMD_TOGGLE_CONTACT_SIDEBAR = 'CMD_TOGGLE_CONTACT_SIDEBAR';
 export const CMD_REOPEN_CONVERSATION = 'CMD_REOPEN_CONVERSATION';
 export const CMD_RESOLVE_CONVERSATION = 'CMD_RESOLVE_CONVERSATION';
 export const CMD_SNOOZE_CONVERSATION = 'CMD_SNOOZE_CONVERSATION';
+export const CMD_CHAT_LIST_SNOOZE_CONVERSATION =
+  'CMD_CHAT_LIST_SNOOZE_CONVERSATION';
+export const CMD_TOGGLE_CONVERSATIONS_SNOOZE =
```

Comment:
I added a separate command bar bus event to work dynamically that ensures the snooze option only appears when you select it from the bulk action dropdown. The snooze option won't be displayed if you access the command bar using the `cmd+k` shortcut from the chat list screen. This ensures that the snooze feature is only visible when relevant.